packs/equipment-effects/effect-beastmasters-sigil-armor-greater.json

```diff
@@ -1,6 +1,6 @@
 {
     "_id": "gZOED4T3o6giterN",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/creatures/reptiles/lizard-iguana-green.webp",
```

Comment:
Did you intend for the greater, major, and normal to have different icons?

Reply:
Different animals per the description, so I was going for that.